fpr/views.py

```diff
@@ -140,27 +138,27 @@ def formatversion_delete(request, format_slug, slug):
     version = get_object_or_404(fprmodels.FormatVersion, slug=slug, format=format)
     dependent_objects = utils.dependent_objects(version)
     breadcrumbs = [
-        {'text': 'Formats', 'link': reverse('format_list')},
+        {'text': _('Format versions'), 'link': reverse('format_list')},
         {'text': format.description, 'link': reverse('format_detail', args=[format.slug])},
     ]
     if request.method == 'POST':
         if 'disable' in request.POST:
             version.enabled = False
-            messages.info(request, 'Disabled.')
+            messages.info(request, _('Disabled.'))
             for obj in dependent_objects:
                 obj['value'].enabled = False
                 obj['value'].save()
         if 'enable' in request.POST:
             version.enabled = True
-            messages.info(request, 'Enabled.')
+            messages.info(request, _('Enabled.'))
         version.save()
         return redirect('format_detail', format.slug)
     return render(request, 'fpr/disable.html',
         context({
             'breadcrumbs': breadcrumbs,
             'dependent_objects': dependent_objects,
             'form_url': reverse('formatversion_delete', args=[format.slug, version.slug]),
-            'model_name': 'Format Version',
```

Comment:
Good catch! I'm changing the button where that attribute `model_name` was included to only show `Disable,Enable` and added an additional heading on the same page (`disable.html`) to make sure that the user sees it:

```html
<h2>
  {% if object.enabled %}
    {% blocktrans with object=object %}Disable {{ object }}{% endblocktrans %}
  {% else %}
    {% blocktrans with object=object %}Enable {{ object }}{% endblocktrans %}
  {% endif %}
</h2>
```

It turns out that accessing to `_meta.verbose_name` from a template requires a custom templatetag, but `__unicode__()` worked.